IsraelHiking.Web/tsconfig.json

```diff
@@ -5,6 +5,7 @@
     "baseUrl": "src",
     "forceConsistentCasingInFileNames": true,
     "noImplicitReturns": true,
+    "noImplicitAny": false,
```

Comment:
How did this help? In general I think the right approach would be to add `noImpicutAny: true` as it would create better code, but I think `"noImplicitAny": false` is the default, so I'm not sure this is needed...?

Reply:
without this I get:

```
Error: src/application/services/route-statistics.service.ts:4:32 - error TS7016: Could not find a declaration file for module 'moving-median'. '.../IsraelHikingMap/Site/IsraelHiking.Web/node_modules/moving-median/median.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/moving-median` if it exists or add a new declaration (.d.ts) file containing `declare module 'moving-median';`

4 import createMedianFilter from "moving-median";
                                 ~~~~~~~~~~~~~~~
```

and if I try `npm i --save-dev @types/moving-median` I get:
```
npm i --save-dev @types/moving-median
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/@types%2fmoving-median - Not found    
npm ERR! 404 
npm ERR! 404  '@types/moving-median@latest' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.
```

https://stackoverflow.com/questions/41462729/typescript-react-could-not-find-a-declaration-file-for-module-react-material

Reply:
Ahh, I see.
Checkout this file, exactly for this:
https://github.com/IsraelHikingMap/Site/blob/master/IsraelHiking.Web/src/application/models/non-typed-modules.d.ts